keps/sig-storage/1682-csi-driver-skip-permission/README.md

```diff
@@ -182,22 +179,19 @@ A test plan will include the following tests:
 
 ### Rollout, Upgrade and Rollback Planning
 
-_This section must be completed when targeting beta graduation to a release._
-
 * **How can a rollout fail? Can it impact already running workloads?**
-  Try to be as paranoid as possible - e.g. what if some components will restart
-  in the middle of rollout?
+  No, a rollout should not impact running workloads, since the default behavior
+  remains the same and a pod spec change is required to use the new field.
```

Comment:
It requires a change in `CSIDriver` object though, so default behaviour remains the same.

Reply:
Thanks, updated.